src/store/modules/wallet/wallet.js

```diff
@@ -91,6 +94,17 @@ export default {
 				commit('setSelectedAddress', account.address)
 			}
 		},
+		registerType({commit,state}, payload) {
+			if (state.activeClient) {
+				if (state.activeClient.registry.lookupType(payload.typeUrl)===undefined) {
```

Comment:
Directly mutating/accessing a private property of the signing client as a workaround for now. Subject to change, see: https://github.com/cosmos/cosmjs/issues/628